src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -1,50 +1,57 @@
 package edu.berkeley.ce.rockslicing
 
-/** A simple tool for parsing command line inputs
-  * @constructor Create a new command reader
-  */
+/**
+ * A simple wrapper for parsing command line inputs
+ */
 object CommandReader {
-  val usage =
-    """
-       Usage: <spark arguments> sparkRocks-assembly-1.0.jar -inputFile <inputFile> -numberSeedJoints <Int> <output opts>
-         where:
-              <spark arguments>    See www.spark.apache.org for details on how to submit Spark applications
-              <inputFile>          File that contains input information on the rock volume of interest and joint sets
-              <Int>                Integer that specifies the number of joints to process before initiating parallel
-                                   instances
-         and <output opts> is one or all of the following
-              -toVTK               Generate output that can be converted to VTK format by rockProcessor.
-                                   Output file format is JSON.
-              -toIE                Generate output that uses inequalities to represent rock blocks along with the
-                                   blocks' center of mass. Output file format is JSON.
-    """.stripMargin
-
-  type OptionMap = Map[Symbol, String]
-
-  def parseArguments(inputs: Array[String]): OptionMap = {
-    if (inputs.length == 0) {
-      println(usage)
-      sys.exit(1)
-    }
+  case class Config(
+    inputFile: String = ".",
+    numSeedJoints: Int = 25,
```

Comment:
Not really. This default value is actually never used because I've told `scopt` that the corresponding command line argument is required, but we have to initialize it to something.